zebra-state/src/service.rs

```diff
@@ -162,7 +163,7 @@ pub(crate) struct StateService {
 
     /// A set of non-finalized block hashes that have been sent to the block write task.
     /// Hashes of blocks below the finalized tip height are periodically pruned.
-    sent_non_finalized_block_hashes: SentHashes,
+    sent_blocks: SentHashes,
```

Comment:
I'm wondering if this field rename also means we need to update its comment?

Reply:
Whoops, fixed in [`b325a4e`](https://github.com/ZcashFoundation/zebra/pull/6335/commits/b325a4eb05335f38420ba34a9e5f132cedd9d2e0).